src/panels/config/zha/zha-group-page.ts

```diff
@@ -20,46 +20,51 @@ import {
   fetchGroupableDevices,
   removeGroups,
   removeMembersFromGroup,
-  ZHADevice,
   ZHAGroup,
+  ZHADeviceEndpoint,
 } from "../../../data/zha";
 import "../../../layouts/hass-error-screen";
 import "../../../layouts/hass-subpage";
 import { HomeAssistant } from "../../../types";
 import "../ha-config-section";
 import { formatAsPaddedHex } from "./functions";
 import "./zha-device-card";
-import "./zha-devices-data-table";
+// eslint-disable-next-line import/no-duplicates
```

Comment:
```suggestion
```

Reply:
doing this broke CI... what am I missing?

Reply:
nvm figured it out